src/detectors/EEMC/EEMC.cc

```diff
@@ -69,6 +69,7 @@ extern "C" {
           "EcalEndcapNIslandProtoClusters", {"EcalEndcapNRecHits"}, {"EcalEndcapNIslandProtoClusters"},
           {
             .adjacencyMatrix = "(abs(row_1 - row_2) + abs(column_1 - column_2)) == 1",
+            .peakNeighbourhoodMatrix = "max(abs(row_1 - row_2), abs(column_1 - column_2)) == 1",
```

Comment:
I'm having a little trouble following what this does: does this matrix define how far apart two maxima need to be in order to be considered two distinct maxima? So here, this would mean that two cells need to be at least more than 1 cell (either in a row or column) away to be considered two maxima?

Reply:
In order for a cell to be considered a peak, it needs to dominate its neighbours. This parameter allows to define what is considered a neighbour.

`(abs(row_1 - row_2) + abs(column_1 - column_2)) == 1` corresponds to
```
 x
xox
 x
```
whereas the `max(abs(row_1 - row_2), abs(column_1 - column_2)) == 1`
```
xxx
xox
xxx
```
Basically, with the new condition, the tower has to dominate 8 neighbouring towers instead of 4, like before.

Reply:
Ahh I see! Makes sense!